cysync/src/pages/mainApp/sidebar/wallet/addAccount/stepperForm.tsx

```diff
@@ -194,13 +194,9 @@ const SendForm: React.FC<StepperProps> = ({ stepsData, handleClose }) => {
   const [activeStep, setActiveStep] = useState(0);
 
   // State for all the data related to the Recipient
-  const [recipientData, setRecipientData] = React.useState<RecipientData>({
-    id: 1,
-    recipient: ' ',
-    amount: '',
-    errorRecipient: '',
-    errorAmount: ''
-  });
+  const [recipientData, addrecipientData] = React.useState<RecipientData[]>([
+    { id: 1, recipient: ' ', amount: '', errorRecipient: '', errorAmount: '' }
+  ]);
```

Comment:
Why's this an array?

Reply:
Changing this into an object causes various cloning issues so debounce hook would use old values and the input field flashes back and forth. I tried using useRef as well still doesn't work this is the best way I could find to fix the issue.